fv3core/stencils/ray_fast.py

```diff
@@ -27,29 +27,28 @@ def dm_layer(rf, dp, wind):
     return (1.0 - rf) * dp * wind
 
 
-@gtstencil()
-def ray_fast_wind(
+def ray_fast_wind_compute(
     u: FloatField,
     v: FloatField,
     w: FloatField,
     dp: FloatFieldK,
     pfull: FloatFieldK,
     dt: float,
     ptop: float,
+    rf_cutoff: float,
```

Comment:
this can still be an external constant rather than a runtime argument (add to externals in the StencilWrapper initialization and then add here`from___externals__ import rf_cutoff`', but since it doesn't help remove any code compile time, perhaps doesn't matter, but I think it does make sense to keep constant namelist values as externals

Reply:
So `rf_cutoff` is also used at realtime to calculate the nudge which depends on another live arg. So I kept it as a class cached var for that, and passed it to the stencils as an external